tests/runner/origin/base_host.py

```
@@ -467,7 +459,7 @@ async def run_genvm(
 	code: bytes | None = None,
 	calldata: bytes,
 	leader_nondet_results: list[bytes] | None = None,
-	message_fee_allocation: list[MessageFeeAllocationNode] = [],
+	message_fee_allocation: list[fees.MessageAllocationNode] = [],
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)